doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
+Dependent resources *within the same namespace as the owning CR* are
+tracked with the `ownerReference` field.
+
+`ownerReference` structure:
+  * apiVersion: {group}/{version}
+  * kind: {kind}
+  * name: {metadata.name}
+  * uid: {metadata.uid}
+
+**Example:**
+
+```yaml
+
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+An `annotation` is used instead of an `ownerReference` if the dependent
+resource is in a different namespace than the CR or the dependent
+resource is a cluster level resource.
+
+`annotation` structure:
+  * operator-sdk/primary-resource: {metadata.namepace}/{metadata.name}
+  * operator-sdk/primary-resource-type: {kind}.{group}
+
+*Note: <group> must be determined by splitting the apiVersion field at the "/"*
+
+```yaml
+metadata:
+  annotations:
+    operator-sdk/primary-resource: default/example-memcached
+    operator-sdk/primary-resource-type: Memcached.cache.example.com
+```
+
+A GET request to the owning resource will provide the necessary data to
+construct an `ownerReference` or an `annotation`.
+
+`$ kubectl get memcacheds.cache.example.com -o yaml`
+
+`kubectl edit` can be used to update the resources by hand.
+
+### Migration Playbook
+
+If you have many resources to update, it may be easier to use the
+following (unsupported) playbook.
+
+#### vars.yml
+
+Users will configure the playbook by providing a `vars.yml` file which will specify:
+  * owning_resource
+      * apiVersion
+      * kind
+      * name
+      * namespace
+  * resources_to_own (list): For each resource, specify:
+      * name
+      * namespace (if applicable)
+      * apiVersion
+      * kind
+
+**Example File:**
+
+```yaml
+owning_resource:
+  apiVersion: cache.example.com/v1alpha1
+  kind: Memcached
+  name: example-memcached
+  namespace: default
+
+resources_to_own:
+  - name: example-memcached-memcached
+    namespace: default
+    apiVersion: apps/v1
+    kind: Deployment
+  - name: example-memcached
+    apiVersion: v1
+    kind: Namespace
+```
+#### playbook.yml
```

Comment:
```suggestion
```suggestion
**<add scallfold path>/playbook.yml**
```
```

Reply:
There is no path or scaffolding for this `playbook.yml`, it's just an example that the user could copy into a file and use as a starting point for doing this migration.

Reply:
The way I wrote this is a little misleading. I'll reword some things to make this clear.